pkg/build/client/clients.go

```diff
@@ -0,0 +1,69 @@
+package client
```

Comment:
I'd propose renaming the file to `client.go`, similar to what we have in `pkg/client/client.go`.